src/goose/console/models/core.clj

```diff
@@ -0,0 +1,7 @@
+(ns goose.console.models.core
+    (:require [goose.console.models.redis.home :refer [jobs-size]]))
+
+;TODO: Need a better way to dispatch UI layer
+(defn dispatch-jobs-stats [broker]
+      (when (= goose.brokers.redis.broker.Redis (type broker))
```

Comment:
What's a good method to dispatch based on the type of the argument?

Reply:
Multi methods are used to dispatch based of type.
We can use multi-methods to dispatch based in broker.